core/new-gui/src/app/dashboard/user/component/user-workflow/ngbd-modal-workflow-executions/workflow-execution-modal.component.ts

```diff
@@ -350,29 +331,24 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit, AfterViewIn
   }
 
   onGroupDelete() {
-    if (this.workflow.wid !== undefined) {
-      this.workflowExecutionsService
-        .groupDeleteWorkflowExecutions(this.workflow.wid, Array.from(this.setOfEid))
-        .pipe(untilDestroyed(this))
-        .subscribe({
-          complete: () => {
-            this.allExecutionEntries = this.allExecutionEntries?.filter(
-              execution => !Array.from(this.setOfExecution).includes(execution)
-            );
-            this.handlePaginationAfterDeletingExecutions();
-            this.setOfEid.clear();
-            this.setOfExecution.clear();
-          },
-        });
-    }
+    this.workflowExecutionsService
```

Comment:
the `if (this.workflow.wid !== undefined)` check is gone. is it intentional? can it be undefined?

Reply:
It cannot be undefined. It's removed on purpose.